addons/web/static/src/core/colorpicker/colorpicker.js

```diff
@@ -523,4 +523,18 @@ export class Colorpicker extends Component {
         this._updateUI();
         this._colorSelected();
     }
+    /**
+     * Called when the hex color input's input event is triggered.
+     *
+     * @private
+     * @param {Event} ev
+     */
+    _onHexColorInput(ev) {
+        const hexColorValue = ev.target.value.replace("#", "");
```

Comment:
As we discussed, could be cool to remove all # in case they copy something while leaving the previous #